deploy/helm-operator-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+---
+  apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: fluxhelmresources.integrations.flux
+  spec:
+    group: integrations.flux
```

Comment:
The advice I have seen for naming this suggests including the FQDN of one's organisation, as well as something representing the group itself. So, I suggest `integrations.flux.weave.works`, or if you want to be really specific, `helm.integrations.flux.weave.works`.

Reply:
To be fair, it was the only advice I found (https://docs.openshift.org/latest/admin_guide/custom_resource_definitions.html)

Reply:
Will change. _helm.integrations.flux.weave.works_ seems good to me (in case we integrate more), even if quite long

Reply:
Yes, I saw that one too. I considered other options, very similar to yours, but could not decide. I thought about the FQDN, but then the naming seemed so long. So I just started with integrations.flux with the prospect of a change (which came :) )

Renamed.